### DISCUSSION TITLE
How to overwrite handler command in template aws-python-docker

### DISCUSSION THREAD

**Discussion by vinkaga** [serverless/serverless#9964] [Q&A] [answered]
I created sample code using `aws-python-docker`. In its `Dockerfile` it says that `app.handler` command can be overwritten from `serverless.yml`. Overwrite functionality is required if 2 separate lambdas are using the same code/docker setup.

My changes are

Added another handler to app.py 
```python
def handler2(event, context):
    body = {
        "message": "Hello, second handler!",
    }
    response = {
        "statusCode": 200,
        "body": json.dumps(body)
    }
    return response
```
 
Modified `serverlesss.yml` to add cmd
```yml
functions:
  hello:
    image:
      name: appimage
      handler: app.handler2
```

On invoke, I still get
```
{
    "statusCode": 200,
    "body": "{\"message\": \"Hello, world! Your function executed successfully!\"}"
}
```
Implying that the `app.handler` was invoked, not `app.handler2`.

How can I invoke `app.handler2` from one lambda and still be able to invoke `app.handler` from another?

**Comment by pgrzesik** [comment]
Hello @vinkaga :wave: There's no such property as `handler` when using `image`, you should see warnings about unrecognized property in your output in such a situation. You should use `image.command` property instead, please see the docs for more details: https://www.serverless.com/framework/docs/providers/aws/guide/functions#referencing-container-image-as-a-target

**Comment by vinkaga** [comment, marked as answer]
I was able to achieve my goal by checking the `context.function_name` value and calling the appropriate handler.

**Reply by pgrzesik** [reply]
That's definitely an option, but you can achieve that just by overriding `image.command` and avoid having extra logic in your handler function :+1:
